index.json

```diff
@@ -1 +1 @@
ame":"safer-eval","PackageVulnerabilityID":"1","CodebasePrimaryLanguage":"JavaScript","VulnerabilityDescription":"Arbitrary Code Execution","Severity":"9.9","AffectedVersionRange":"*","DisclosureDate":"2020-02-21","Bounty":{"Credit":990,"Cash":25}},{"Registry":"npm","PackageName":"strider-grit","PackageVulnerabilityID":"1","CodebasePrimaryLanguage":"JavaScript","VulnerabilityDescription":"Remote Code Execution","Severity":"7.8","AffectedVersionRange":"*","DisclosureDate":"2020-06-16","Bounty":{"Credit":780,"Cash":25}},{"Registry":"npm","PackageName":"windows-edge","PackageVulnerabilityID":"1","CodebasePrimaryLanguage":"JavaScript","VulnerabilityDescription":"Arbitrary Code Execution","Severity":"7.3","AffectedVersionRange":"*","DisclosureDate":"2020-06-16","Bounty":{"Credit":990,"Cash":25}},{"Registry":"pip","PackageName":"cinder","PackageVulnerabilityID":"1","CodebasePrimaryLanguage":"Python","VulnerabilityDescription":"Insufficiently Protected Credentials","Severity":"6.5","AffectedVersionRange":"*","DisclosureDate":"2020-06-09","Bounty":{"Credit":650,"Cash":25}},{"Registry":"rubygems","PackageName":"em-imap","PackageVulnerabilityID":"1","CodebasePrimaryLanguage":"Ruby","VulnerabilityDescription":"Man-in-the-Middle","Severity":"8.2","AffectedVersionRange":"*","DisclosureDate":"2020-05-27","Bounty":{"Credit":820,"Cash":25}},{"Registry":"rubygems","PackageName":"goliath","PackageVulnerabilityID":"1","CodebasePrimaryLanguage":"Ruby","VulnerabilityDescription":"HTTP Request Smuggling","Severity":"5.9","AffectedVersionRange":"*","DisclosureDate":"2020-06-09","Bounty":{"Credit":590,"Cash":25}},{"Registry":"rubygems","PackageName":"radiant","PackageVulnerabilityID":"1","CodebasePrimaryLanguage":"Ruby","VulnerabilityDescription":"Cross-site Scripting (XSS)","Severity":"5.4","AffectedVersionRange":"*","DisclosureDate":"2020-02-13","Bounty":{"Credit":590,"Cash":25}},{"Registry":"packagist","PackageName":"teampass","PackageVulnerabilityID":"1","CodebasePrimaryLanguage":"PHP","VulnerabilityDescription":"Unauthorized File Access","Severity":"7.5","AffectedVersionRange":"*","DisclosureDate":"2020-04-29","Bounty":{"Credit":750,"Cash":25}},{"Registry":"maven","PackageName":"argonaut","PackageVulnerabilityID":"1","CodebasePrimaryLanguage":"Java","VulnerabilityDescription":"Denial of Service (DoS)","Severity":"4.9","AffectedVersionRange":"*","DisclosureDate":"2020-04-10","Bounty":{"Credit":490,"Cash":25}},{"Registry":"maven","PackageName":"openid-connect-java-spring-server","PackageVulnerabilityID":"1","CodebasePrimaryLanguage":"Java","VulnerabilityDescription":"Cross Site Scripting (XSS)","Severity":"6.1","AffectedVersionRange":"*","DisclosureDate":"2020-02-03","Bounty":{"Credit":610,"Cash":25}},{"Registry":"maven","PackageName":"gwtupload","PackageVulnerabilityID":"1","CodebasePrimaryLanguage":"Java","VulnerabilityDescription":"Denial of Service (DoS)","Severity":"7.5","AffectedVersionRange":"*","DisclosureDate":"2020-06-01","Bounty":{"Credit":750,"Cash":25}}]
```

Comment:
I would be careful merging this into master as a lot of the data is incorrect and potentially may break existing bounties.